app/src/main/java/ar/com/wolox/android/example/ui/home/news/individualNew/IndividualNewFragment.kt

```diff
@@ -0,0 +1,68 @@
+package ar.com.wolox.android.example.ui.home.news.individualNew
+
+import android.net.Uri
+import android.os.Bundle
+import android.widget.Toast
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.network.News
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_individual_new.*
+import javax.inject.Inject
+
+class IndividualNewFragment @Inject constructor() : WolmoFragment<IndividualNewPresenter>(), IIndividualNewView {
+
+    override fun layout(): Int = R.layout.fragment_individual_new
+
+    override fun init() {
+        val id = arguments?.getParcelable<News>(NEW_ID)?.id
```

Comment:
Use `arguments?.getParcelable<News>(NEW_ID)?.run (or let) {` to avoid repeating the get of the argument.